usr/lib/cca_stdll/cca_specific.c

```diff
@@ -1628,6 +1750,19 @@ static CK_RV cca_get_adapter_domain_selection_infos(STDLL_TokData_t *tokdata)
                 &rule_array_count, rule_array,
                 &verb_data_length, (unsigned char *)&cca_private->num_domains);
 
+   /*
```

Comment:
Instead of trying the query and catching the error (that will always be returned on AIX) you could ifdef out the query in case of AIX and always preset the num_usagedoms ans num_domains fields instead.